src/main/frontend/util.cljc

```diff
@@ -1186,7 +1186,9 @@
 #?(:cljs
    (defn zh-CN-supported?
      []
-     (contains? (set (system-locales)) "zh-CN")))
+     (let [system-locales (set (system-locales))]
+       (or (contains? system-locales "zh-CN")
+           (contains? system-locales "zh-Hans-CN")))))
```

Comment:
Better check against ending with `-CN` or contains `-CN-`

Reply:
Since this condition is used to check country/region-specific websites.

- to check all variants of Chinese(any country or region): starts with `zh-`
- to check region of mainland China: ends with`-CN` or contains `-CN-`